local-cluster/src/local_cluster.rs

```diff
@@ -201,11 +208,6 @@ impl LocalCluster {
                 validator_config.staked_nodes_overrides = Arc::new(RwLock::new(overrides));
             }
 
-            assert!(
```

Comment:
Why kill this assert?

Reply:
This assert is useless as the branch condition already cover it.

Reply:
see "if config.tpu_use_quic"